app/controllers/quick_search/search_controller.rb

```diff
@@ -7,18 +7,41 @@ class SearchController < ApplicationController
     include QuickSearch::EncodeUtf8
     include QuickSearch::QueryFilter
     include QuickSearch::SearcherConfig
+    include QuickSearch::SearchHelper
 
     require 'benchmark_logger'
 
     before_action :doi_trap, :log_query
     after_action :realtime_message, only: [:index]
-
+    helper_method :all_good_bets
     def index
       loaded_searches
       @common_searches = common_searches
       http_search
     end
 
+    def all_good_bets
+      good_bets = []
+      best_bet_links = !@best_bets.is_a?(QuickSearch::SearcherError) ? @best_bets.results.flatten.map{|elem|elem[:link] ? stripcharacter(elem[:link], '/') : elem[:url] ? stripcharacter(elem[:url], '/') : ''} : []
```

Comment:
I would caution against using multiple ternary statements in one line (or too many method calls). It is hard to figure out what is going on without spending a couple mins looking at it. Splitting these out would help make it more readable. A comment would also help if the line has a lot going on.